README.md

```diff
@@ -142,21 +143,7 @@ the `voxel51/vehicle-sense` analytic:
     "analytic": "voxel51/vehicle-sense",
     "version": "0.3",
     "job_id": "2ffe1110-b446-427d-8829-db9ac95d0638",
-    "inputs": {
```

Comment:
So, how do we want to handle this? We're still, in fact, populating these fields in the TaskConfig, to maintain backwards compatibility with old analytic versions. And, the SDK as written will silently fallback to using these fields if it fails to get a fresh URL from the Platform.

So, do we leave these fields in here, and then add some explanation below that explains the two-stage procedure? Or move that explanation to the bottom of the README or some "compatibility" section? It should be explained somewhere.

Reply:
https://github.com/voxel51/platform-sdk/pull/25#issuecomment-580825193
Are the SDK docs the right place to explain that? The goal here is to provide a more reliable SDK without requiring users of it to change anything. Information about what exactly the task JSON contains would be relevant to people developing without using the SDK, but I'm not sure where/if there is documentation for that.

Reply:
Fair enough. In that case we should just remove the TaskConfig JSON example altogether. It's not providing any value because (a) it's not actually the correct JSON, and (b) there are a variety of other aspects of the interface that are not described here at all, so a partially detailed explanation doesn't add a lot of value.

I originally showed the JSON because it would enable someone who didn't want to use the SDK (e.g., if they are writing in another language) to grok the format and where they're supposed to read/write data from, and just build it all themselves. Now it's not so simple (and, anyway, the README doesn't explain the API routes to use to post job statuses, etc) so, as you say, there's no value in describing in that much detail in the README

Reply:
Sounds good. (It may be worth documenting this somewhere, eventually, maybe in a separate doc or something?)